### ISSUE TITLE
Should loading and error messages trap focus?

### ISSUE THREAD

**Issue by aslagle** [NYPL-Simplified/webpub-viewer#99]

**Comment by seanredmond** [comment]
Yes, but you have to be careful about it. Here are some articles that discuss the issues:

- https://www.marcozehe.de/2015/02/05/advanced-aria-tip-2-accessible-modal-dialogs/
- http://blog.barrierbreak.com/2017/02/14/modal-dialogs-and-accessibility-a-tricky-minefield/
